templates/firefox-ios.xliff

```diff
@@ -2635,11 +2626,11 @@
         <note>Title for the second  panel 'Search' in the First Run tour.</note>
       </trans-unit>
       <trans-unit id="Intro.Slides.Sync.Description">
-        <source>Use Sync to find the bookmarks, passwords, and other things you save to Firefox on all your devices.</source>
+        <source>Get your bookmarks, history and passwords on all your devices.</source>
```

Comment:
This is a big enough change in the meaning of the string that it should get a new string ID in order to be marked by our tools as needing to get an updated translation